app/controllers/users_controller.rb

```diff
@@ -0,0 +1,32 @@
+class UsersController < ApplicationController
+  def show
+    @user = User.find_by id: params[:id]
```

Comment:
Khi tìm không thấy thì xử lý thến nào em

Reply:
- 1 là trả về không tìm thấy (đối với thao tác tìm kiếm)
- 2 là nó sẽ trả về trang sign up hoặc sign in

Reply:
Thế ở đây là em đang xử lý theo trường hợp nào đó